common/util/cmd_positional_arguments.h

```diff
@@ -0,0 +1,67 @@
+// Copyright 2017-2020 The Verible Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef VERIBLE_COMMON_UTIL_CMD_POSITIONAL_ARGUMENTS_H_
+#define VERIBLE_COMMON_UTIL_CMD_POSITIONAL_ARGUMENTS_H_
+
+#include <string>
+#include <vector>
+
+#include "absl/status/status.h"
+#include "absl/strings/string_view.h"
+
+namespace verible {
+
+class CmdPositionalArguments {
+ public:
+  explicit CmdPositionalArguments(const std::vector<char *> &args)
+      : all_args_(args){};
+
+  // Main function that parses arguments.
+  absl::Status ParseArgs();
+
+  // Gets include directories arguments.
+  std::vector<absl::string_view> GetIncludeDirs();
```

Comment:
Given that the GetIncludeDirs(), GetDefines() and GetFiles() only return a member, implementing them directly in the header is probably improving readability. No need to have a multi-line implementation in the *.cc file, when we can write directly here

```c++
const std::vector<absl::string_view>& GetIncludeDirs() const { return defines_; }
```

Also
  * The return value should be a const reference, as we don't want to copy the full vector.
  * The method itself should be const qualified (the `const` at the end), as this does not change the state of the object.

(but also see comment above: in favor of parsing everyhing into a `FileList` struct)